src/app/host/[roomId]/page.tsx

```diff
@@ -202,16 +239,17 @@ export default function HostRoom({ params }: { params: { roomId: string } }) {
 					roomUrl={roomUrl}
 					allowedPointsEmitter={allowedPointsEmitter}
 					defaultStoryPointValues={defaultStoryPointValues}
+					allowedStoryPoints={allowedStoryPoints}
 					setAllowedStoryPoints={setAllowedStoryPoints}
 				/>
 			</div>
 
-			{/* TODO: Remove when development is done */}
-			<div className='w-32 absolute top-2 left-8'>
-				<Link href='/host' className='underline text-sky-500 hover:text-sky-300 text-sm'>
-					Create Room
-				</Link>
-			</div>
+			{/* TODO Remove HostDevButtons when development is done */}
+			<HostDevButtons
+				handleAddRandomUsers={handleAddRandomUsers}
+				handleRemoveRandomUsers={handleRemoveRandomUsers}
```

Comment:
suggestion: a prop convention for callback is to call `on*`, so

```suggestion
			<HostDevButtons
				onAddRandomUsers={handleAddRandomUsers}
				onRemoveRandomUsers={handleRemoveRandomUsers}
```

Reply:
q: But wait... why did you decide to put these handlers in the parent, instead of inside the component? If it's a devtool component, no matter where we call it in the app, it should simply work, no?

Reply:
I put them in the parent because all the state is in the parent:  realUsers & allUsersPointsData. Plus the emitter function they need is in the parent as well. It made more sense to pass the 2 handlers to the child component vs passing the 2 state setters to the child and doing another emitter in the child. 2 props vs 2 props and an emitter. 

I think it is best where it is, but if you think I should move it, I can.

Reply:
Made the change to 'on' vs 'handle'

Reply:
I see what you mean, but you can still reuse both. Think about reusability and "separation of concerns". Or think about this: the day you decide to reuse that DevButtons component somewhere, which files do you need to touch? As it is, you'll need to call the component, and duplicate all the logic of adding/removing the users (the handlers). That's a clear hint that that logic should be somewhere else.

Even with the state in the parent, nothing stops you from passing it as prop. The only thing left then is the emitter `allUsersPointsEmitter`. That you can call inside the component too. Or, you can do it in the parent, as it's just one line. Like this:

```js
	<HostDevButtons
		allUsersPoints={allUsersPointsData}
		onSetAllUsersPoints={(newUsersPoints) => {
			allUsersPointsEmitter(newUsersPoints)
			setAllUsersPointsData(newUsersPoints)
		}}
	/>
```

Reply:
Okay, that makes a lot of sense. I hadn't thought about it that way. I'll move those handlers to the component.

I really like this pattern of passing functions to other components. That is something I have not done in the past, but I'm seeing how effective it is. That is a big takeaway from this project!! I'll make those changes in the next commit.

Reply:
I ended up passing the state, setState, and emitter to the HostDevButtons components. Everything worked as expected.

Question:  since those 2 handlers aren't callback anymore, should they still be named with "on"? Or should they now be named with "handle" since they are handlers for the buttons?

I committed the changes. DONE!

Reply:
A function can always be named on* because it’s called when something happens ;)